lib/lexer.js

```diff
@@ -133,24 +141,34 @@ Lexer.prototype._output = function() {
     text = this.source.substring(this._pos, i);
     this._pos = i;
   }
-  
+
   this._line += text.split("\n").length - 1;
   this._newlineBefore = text.indexOf("\n") >= 0;
   return this._token(Token.OUTPUT, text, true);
 }
 
 Lexer.prototype._embed = function() {
   var tag = this._next() + this._next()
-    , ch = this._peek();
+    , ch = this._peek()
+    ;
+
+  this._stripNewLine = false;
```

Comment:
It might be good to initialize this in `tokenize()` and also reset to `false` at the end of `_skipNewline()`, instead of here—just to make sure you're not skipping whitespace you don't intend outside of the embed.

That said, this lexer doesn't actually properly retain newlines and other whitespace, so it won't properly render templates that expect significant whitespace. That's fine for HTML and other languages that don't care about whitespace, but is problematic for `text/plain` and other white-space sensitive formats. That's an issue that should be resolved another time.